### DISCUSSION TITLE
SecOnion Distro Deploy Sensor Install repo issues

### DISCUSSION THREAD

**Discussion by Director-Fusion** [Security-Onion-Solutions/securityonion#13485] [2.4]
### Version

2.4.90

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

32gb

### Storage for /

?

### Storage for /nsm

?

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Anyone running into issues installing a Sensor on a distributed deployment? If I do an IP address seems to work fine, however, I am going to setup this up as a remote sensor so when I use fqdn. Sensor install will fail with... "Failed to download metadata for repo 'securityonion': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried". I did try updating repo manually to no luck. Any ideas? Or broken repo.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment]
Sounds like there may be a communication issue between the Sensor and the Manager. Are all of the ports listed here open? You can check them with "nc -vz [Manager IP] [port number]" from the Sensor's command line.

https://docs.securityonion.net/en/2.4/firewall.html#node-communication

**Reply by Director-Fusion** [reply]
Not quite sure what the root cause was. I did as you said and connected just fine. I ended up reinstalling SecOnion from scratch and worked just fine on bare metal. I also noticed issues with sensor and heavynode setups if the monitor interface was in promisc mode prior to enrolling the sensors/heavys.
